db/migrations/manual/20201202-cclf-pre-mbi.sql

```diff
@@ -0,0 +1,7 @@
+-- Script will remove CCLF data that existed before MBIs were reliable
```

Comment:
What is the reason for requiring these scripts be run manually instead of by using our standard migration strategy?  Is it because the rollback is not possible through automation (i.e., by using `up` and `down`)?

Reply:
Yup that's exactly right, there's no ability for us to rollback these removed entries.